dblp/python/superuser-init.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/python
+
+"""
+This is the initialization script for dblp.org
+
+It should only be run ONCE to kick off the conference. It can only be run by the Super User.
+
+"""
+
+## Import statements
+import argparse
+import csv
+import sys
+from openreview import *
+
+## Handle the arguments
+parser = argparse.ArgumentParser()
+parser.add_argument('--baseurl', help="base URL")
+parser.add_argument('--overwrite', help="If set to true, overwrites existing groups")
+parser.add_argument('--username')
+parser.add_argument('--password')
+
+args = parser.parse_args()
+
+## Initialize the client library with username and password
+if args.username!=None and args.password!=None:
+    openreview = Client(baseurl=args.baseurl, username=args.username, password=args.password)
+else:
+    openreview = Client(baseurl=args.baseurl)
+
+groups = []
+overwrite = True if (args.overwrite!=None and args.overwrite.lower()=='true') else False
+def overwrite_allowed(groupid):
+    if not openreview.exists(groupid) or overwrite==True:
+        return True
+    else:
+        return False
+
+if openreview.user['id'].lower()=='openreview.net':
+    
+    #########################
+    ##    SETUP GROUPS     ##
+    ######################### 
+
+    if overwrite_allowed('dblp.org'):
+        iclr            = Group('dblp.org',      
+            readers     = ['OpenReview.net'], 
+            writers     = ['OpenReview.net','dblp.org'], 
+            signatures  = ['OpenReview.net'], 
+            signatories = ['dblp.org'], 
+            members     = ['spector@cs.umass.edu'] )
+        groups.append(iclr)
+        ## Post the groups
+    
+    for g in groups:
+        print "Posting group: ",g.id
+        openreview.post_group(g)
+
+
+
+
+
+
+    reply = {
+        'forum': None,
+        'replyto': None,
+        'readers': {
+            'description': 'The users who will be allowed to read the above content.',
+            'values': ['everyone']
+        },
+        'signatures': {
```

Comment:
signatures and writes will be DBLP.org/upload, I think we should create this group too,